### PR TITLE
Bug Fix for Resume Button  (petition view) and View Button (index view of petitions)

### PR DESCRIPTION
**Pull request by lionick**

### CODE REVIEW

File: app/View/CoPetitions/index.ctp [hunk 1]
```diff
@@ -232,17 +232,13 @@ if(empty($this->params['named']['op'])) {
...
-            if(!empty($this->params['named']['op']) && $this->params['named']['op'] == 'resume' || $permissions['edit']) {
-              print $this->Html->link(_txt('op.view'),
-                                      array('controller' => 'co_petitions',
-                                            'action' => 'view',
-                                            $p['CoPetition']['id']),
-                                      array('class' => 'editbutton',
-                                      'title' => _txt('op.view-a',array($displayNameWithId)),
-                                      'aria-label' => _txt('op.view-a',array($displayNameWithId)))) . "\n";
-            }
+            print $this->Html->link(_txt('op.view'),
```
Comment (NicolasLiampotis) [inline app/View/CoPetitions/index.ctp:235]: Is there a permission that can be checked before showing the View button?

Comment (lionick) [reply app/View/CoPetitions/index.ctp:235]: The permission for the index page is:
```
 $p['index'] = ($roles['cmadmin']
                   || $roles['coadmin'] || $roles['couadmin']
                   // Only allow "any admin" if org identities are pooled and
                   // we don't have a CO Person ID for the user (so we're in an
                   // Org Identity context, such as search by Org Identity ID)
                   || ($pool
                       && !$roles['copersonid']
                       && ($roles['admin'] || $roles['subadmin']))
                   || $this->Role->isApprover($roles['copersonid']));
if(!empty($this->params['named']['op']) && $this->params['named']['op'] == 'resume') {
      $p['index'] = $p['index'] || $roles['user'];
}
```
in combination with the new feature we add that a couadmin can see only the petitions of his own cous then this button can be shown without any other restrictions.

**Review by ioigoume — APPROVED** [review]
LGTM

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**